tests/e2e/lib.sh

```diff
@@ -439,6 +439,45 @@ setup_automation_flavor_e2e_cluster() {
     fi
 }
 
+# When working as expected it takes less than one minute for the API server to
+# reach ready. Often times out on OSD. If this call fails in CI we need to
+# identify the source of pull/scheduling latency, request throttling, etc.
+# I tried increasing the timeout from 5m to 20m for OSD but it did not help.
```

Comment:
I see you set the wait timeout to be 5 mins. Do you know why did OSD take so long to reinstate? How long does it usually take?

Reply:
@c-du I literally copied the top half of wait_for_api where it checks to make sure the deployment is running.  That includes the copying of the comment.  So I don't know.